lib/logsHandler.ts

```diff
@@ -1,13 +1,19 @@
-import { appendFileSync } from "fs";
-import { tmpdir } from "os";
-import { join } from "path";
-import moment from "moment";
+import {appendFileSync} from "fs";
+import {tmpdir} from "os";
+import {join} from "path";
+import moment from 'moment';
 
-const LOG_FILE_NAME = `installer-${moment(new Date()).format(
-  "YYYY-MM-DDThmmss"
-)}.log`;
-export const LOG_PATH = join(tmpdir(), LOG_FILE_NAME);
+export class Logger {
+    name: string;
+    pathToLogFile: string;
+    
+    constructor(name?: string) {
+        this.name = `${name || "installer"}-${moment(new Date()).format('YYYY-MM-DDThmmss')}.log`
+        this.pathToLogFile = join(tmpdir(), this.name)
```

Comment:
`this.name` is not really needed, just the `filepath` attribute is used (internally)
refactor this